subsys/pm/state.c

```diff
@@ -39,23 +39,21 @@ BUILD_ASSERT(DT_NODE_EXISTS(DT_PATH(cpus)),
 /* Check that all power states are consistent */
 DT_FOREACH_CHILD(DT_PATH(cpus), CHECK_POWER_STATES_CONSISTENCY)
 
-#define NUM_CPU_STATES(n) DT_NUM_CPU_POWER_STATES(n),
```

Comment:
Looks like DT_NUM_CPU_POWER_STATES was added by you in 7bfd0976aad, with a potential move to devicetree.h in the future. Gentle reminder that this is still an abuse of the DT_ namespace.

Reply:
thanks for the reminder here, will likely move that macro to DT API